test/index.js

```diff
@@ -42,7 +42,7 @@ test('Fixtures', function(t) {
 
     try {
       config = JSON.parse(read(join(filepath, 'config.json')))
-    } catch (error) {
+    } catch {
```

Comment:
Did xo do this, if so, I suggest turning that rule off 😞? It’ll break a lot of people I think, no need to do that.

Reply:
It was quite annoying, xo and prettier were fighting between `error`, `_error`, and this.

I can investigate adding `engines` to `package.json`, or similar, which might prevent this kind of change? Otherwise I'll just disable the relevant rules